bin/database-migration/migration_validation.py

```diff
@@ -91,6 +93,29 @@ def get_last_row(cursor, table_name):
     return cursor.fetchone()
 
 
+def check_table_data(error_list):
```

Comment:
Does this need the error_list if it's returning it?
Maybe have it just return the error_list and then append it to your errors after you call it